src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/conf/ConfReaderUtils.java

```diff
@@ -0,0 +1,94 @@
+package org.jenkinsci.plugins.pipeline.utility.steps.conf;
+
+import hudson.FilePath;
+import hudson.model.TaskListener;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.io.StringReader;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.TreeMap;
+
+import static org.apache.commons.lang.StringUtils.isBlank;
+
+public class ConfReaderUtils {
+    static Map<String,String> toStringMap(@Nonnull Map<String,Object> inMap) {
+        Map<String,String> result = new TreeMap<>();
+        for (Map.Entry<String,Object> e : inMap.entrySet()) {
+            result.put(e.getKey(), e.getValue().toString());
```

Comment:
`e.getValue()` could return `null`. Use `String.valueOf(e.getValue())` instead, unless that returns the string `"null"`:)